components/Dialogs/src/MDCAlertController.m

```diff
@@ -369,17 +369,18 @@ - (void)viewDidLoad {
 
 - (void)setupAlertView {
   // Explicitly overwrite the view default if true
-  if (_mdc_adjustsFontForContentSizeCategory) {
-    self.alertView.mdc_adjustsFontForContentSizeCategory = YES;
-  }
   self.alertView.titleLabel.text = self.title;
   self.alertView.messageLabel.text = self.message;
   self.alertView.titleFont = self.titleFont;
   self.alertView.messageFont = self.messageFont;
   self.alertView.titleColor = self.titleColor;
   self.alertView.messageColor = self.messageColor;
+  self.alertView.buttonFont = self.buttonFont;  // b/117717380: Will be deprecated
```

Comment:
Please update this to a GitHub issue url instead.

Reply:
And please use a `TODO` so it can be tracked more easily in the code.